scripts/backup.sh

```diff
@@ -18,9 +18,5 @@ echo "backup created at $FILE_PATH"
 
 if [ "${DELETE_OLD_BACKUPS}" = true ]; then
         echo "removing backups older than ${OLD_BACKUP_DAYS} days"
-        if [ "$(id -u)" -eq 0 ]; then
-                su steam -c "find /palworld/backups/ -mindepth 1 -maxdepth 1 -mtime "+${OLD_BACKUP_DAYS}" -type f -name 'palworld-save-*.tar.gz' -print -delete"
-        else
-                find /palworld/backups/ -mindepth 1 -maxdepth 1 -mtime "+${OLD_BACKUP_DAYS}" -type f -name 'palworld-save-*.tar.gz' -print -delete
-        fi
```

Comment:
I had added deletion only as steam but that didn't protect files owned by others or with write permissions.